src/analyses/libraryFunctions.ml

```diff
@@ -193,7 +193,9 @@ let invalidate_actions = ref [
     "_spin_lock_irqsave", readsAll;(*safe*)
     "_spin_unlock_irqrestore", readsAll;(*safe*)
     "pthread_mutex_init", readsAll;(*safe*)
-    "pthread_mutex_destroy", readsAll;(*safe*)
+    "pthread_mutex_init", readsAll;(*safe*)
```

Comment:
Now `pthread_mutex_init` is here twice and `pthread_mutex_destroy` isn't at all?

Reply:
yup, that was a stupid typo